src/style.css

```diff
@@ -0,0 +1,61 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+* {
+  box-sizing: border-box;
+  font-weight: 300;
+  font-family: "Avenir-Book", sans-serif;
+}
```

Comment:
do u need FF(font-family) for <script>? IMO, no.. let's add FF and font-weight to html tag